hack/ensure-acr-cleanup-schedule.sh

```diff
@@ -18,10 +18,14 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-e2e_registry="ci-e2e/cluster-api-azure-controller-amd64"
+e2e_registry="ci-e2e"
```

Comment:
@devigned this wasn't working 

```
Deleting tags for repository: ci-e2e/cluster-api-azure-controller-amd64
ci-e2e/cluster-api-azure-controller-amd64 repository not found
```

I think this should fix it?

Reply:
I think that was one of the repos at one point. I believe there's a bifurcation in our CI scripts which causes `ci-e2e` to not be added to some of the repos. I think that repository existed at one point. 

We should have a clean up task on each of the repos in the CI ACR instance.

Here's a list of the existing repos:
![image](https://user-images.githubusercontent.com/386473/115675250-6d1d9880-a31c-11eb-82f0-afe2745c3071.png)

Reply:
Some of these are manual tests that @chewong ran in #1264, we should add the ones are going to be created by CI but the ones that were a one time test we can probably delete manually.

I'm not sure why e2e is creating cluster-api-azure-controller-amd64 instead of ci-e2e/cluster-api-azure-controller-amd64, I'll look into it

Reply:
ok, all the other ones are gone. Still don't know why it's using cluster-api-azure-controller-amd64 instead of ci-e2e/cluster-api-azure-controller-amd64

Reply:
Is it because of the REGISTRY environment variable in https://github.com/kubernetes/test-infra/blob/master/config/prow/config.yaml#L721-L725?

Reply:
Ah you are correct...

Reply:
Let's standardize it by changing it to `capzci.azurecr.io/ci-e2e`